streaming/rdm_atomic.c

```diff
@@ -75,9 +75,23 @@ static const char* get_fi_op_name(enum fi_op op)
 	switch (op) {
 	case FI_MIN: return "min";
 	case FI_MAX: return "max";
+	case FI_SUM: return "sum";
+	case FI_PROD: return "prod";
+	case FI_LOR: return "lor";
+	case FI_LAND: return "land";
+	case FI_BOR: return "bor";
+	case FI_BAND: return "band";
+	case FI_LXOR: return "lxor";
+	case FI_BXOR: return "bxor";
 	case FI_ATOMIC_READ: return "read";
 	case FI_ATOMIC_WRITE: return "write";
 	case FI_CSWAP: return "cswap";
+	case FI_CSWAP_NE: return "cswap_ne";
+	case FI_CSWAP_LE: return "cswap_le";
+	case FI_CSWAP_LT: return "cswap_lt";
+	case FI_CSWAP_GE: return "cswap_ge";
+	case FI_CSWAP_GT: return "cswap_gt";
+	case FI_MSWAP: return "mswap";
```

Comment:
Can you see if fi_tostr() handles these?  If not, it should be expanded to do so.